homeassistant/components/hvv_departures/sensor.py

```diff
@@ -61,7 +60,7 @@ def __init__(self, hass, config_entry, session, hub):
     async def async_update(self, **kwargs):
         """Update the sensor."""
 
-        departure_time = utcnow() + timedelta(
+        departure_time = datetime.now() + timedelta(
```

Comment:
The state should be UTC time. If the api needs local time we need to do a separate conversion.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes